packages/shell-api/src/helpers.ts

```diff
@@ -472,11 +491,12 @@ export async function getPrintableShardStatus(
                 },
               },
             ])
-          ).toArray()
+          ).toArray()) as MigrationResult[]
         );
 
-        const migrationsRes: Record<number, string> = {};
-        migrations.forEach((x: any) => {
+        const migrationsRes: ShardingStatusResult['balancer']['Migration Results for the last 24 hours'] =
+          {};
+        migrations.forEach((x) => {
```

Comment:
Just a note for the future, but if you run into existing `.forEach()` into the future, don't be shy to refactor them to `for ... of` 🙂